### DISCUSSION TITLE
Generate SSH profiles from custom host list

### DISCUSSION THREAD

**Discussion by szabosa** [Eugeny/tabby#6503] [Q&A]
Hello Guys,

Did anyone try to generate ssh profiles to Tabby config from a custom host list? I manage 700+ hosts, and of course I don't want to create all connections one by one. I can reach them through jump servers, so basically most of them are the same, except the host name and the ip ofc. I have a list which contains all of them, so my plan is to check in the cfg how does it look like, create a template and generate all ssh connections with a for loop from the custom host list.

Is there anyone who already did a thing like this and can give an example or some advices?
Is that possible to do it at all?

Thanks,
Alex

**Comment by Eugeny** [comment]
You definitely can generate a config snippet from a host list and paste it into the config editor. The config entries generated by Tabby itself are pretty verbose, but most options there aren't required.
